src/parser/jobs/pct/modules/Gauge.tsx

```diff
@@ -240,17 +250,81 @@ export class Gauge extends CoreGauge {
 
 		this.addEventHook(playerFilter.type('action').action(this.data.actions.COMET_IN_BLACK.id), this.onBlackPaintSpender)
 
-		this.addEventHook(playerFilter.type('action').action(this.data.actions.SUBTRACTIVE_PALLETTE.id), this.onSubtractivePalette)
+		this.addEventHook(playerFilter.type('action').action(this.data.actions.SUBTRACTIVE_PALETTE.id), this.onSubtractivePalette)
 
 		const canvasActions = Array.from(this.canvasModifiers.keys())
 		this.addEventHook(playerFilter.type('action').action(oneOf(canvasActions)), this.onCanvasModifier)
 
 		this.addEventHook(playerFilter.type('action').action(oneOf(this.depictionModifiers)), this.onDepictionModifier)
 
-		this.addEventHook(playerFilter.type('action').action(oneOf(this.portraitModifiers)), this.onPortraitModifier)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.portraitGenerators)), this.onPortraitGenerator)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.portraitSpenders)), () => this.portraitGauge.reset())
 
 		// Default assume the player painted before the pull
 		this.canvasGauge.set([CREATURE_MOTIF, WEAPON_MOTIF, LANDSCAPE_MOTIF])
+
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onComplete() {
+		// Suggestion to not overcap the palette gauge
+		const lostSubtractives = Math.floor(this.paletteGauge.overCap / SUBTRACTIVE_COST)
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SUBTRACTIVE_PALETTE.icon,
+			value: lostSubtractives,
+			content: <Trans id="pct.gauge.suggestions.lost-subtractives.content">Overcapping the Palette gauge will cause you to lose uses of <DataLink action="SUBTRACTIVE_PALETTE"/> over the course of the fight, leading to lost potency. Make sure to use your gauge before it overcaps.</Trans>,
+			why: <Trans id="pct.gauge.suggestions.lost-subtractives.why">You lose <Plural value={lostSubtractives} one="# use" other="# uses" /> of <DataLink showIcon={false} action="SUBTRACTIVE_PALETTE" /> due to overcapping the Palette gauge.</Trans>,
```

Comment:
```suggestion
			why: <Trans id="pct.gauge.suggestions.lost-subtractives.why">You lost <Plural value={lostSubtractives} one="# use" other="# uses" /> of <DataLink showIcon={false} action="SUBTRACTIVE_PALETTE" /> due to overcapping the Palette gauge.</Trans>,
```
_Nit:_ This should be the same past tense as other suggestions

Reply:
Wops. Good catch, is fixed.